hikari/impl/bot.py

```diff
@@ -59,6 +59,7 @@
 from hikari.utilities import date
 from hikari.utilities import event_stream
 from hikari.utilities import ux
```

Comment:
please run isort if you havent already

I would assume `ux` comes after `un`, but might be wrong